test/__tests__/__snapshots__/early-exit.js.snap

```diff
@@ -0,0 +1,165 @@
+    --no-ignore               Do not look for an ignore file
+    --ignore-path <path...>   Path to a file with patterns describing files to ignore
+                              Multiple values are accepted
+                              Defaults to [.gitignore, .prettierignore]
+    --plugin <package...>     Add a plugin
+                              Multiple plugins are accepted
+                              Defaults to []
+    --with-node-modules       Process files inside the "node_modules" directory
+    
+  EDITOR OPTIONS
+    
+    --cursor-offset <int>     Print (to stderr) where a cursor at the given position would move to after formatting
+                              Defaults to "-1"
+    --range-end <int>         Format code ending at a given character offset (exclusive)
+                              The range will extend forwards to the end of the selected statement
+                              Defaults to "Infinity"
+    --range-start <int>       Format code starting at a given character offset
+                              The range will extend backwards to the start of the first line containing the selected statement
+                              Defaults to "0"
+    
+  OTHER OPTIONS
+    
+    --help                    Display help for the command
+    --version, -v             Display the version number
+    --no-cache                Do not use the built-in caching mechanism
+    --cache-location <path>   Path to the cache file
+    --no-color                Do not colorize output messages
+    --no-error-on-unmatched-pattern 
+                              Prevent errors when pattern is unmatched
+    --ignore-unknown, -u      Ignore unknown files
+    --insert-pragma           Insert @format pragma into file's first docblock comment
+                              Defaults to "false"
+    --log-level <silent|error|warn|log|debug>
+                              What level of logs to report
+                              Defaults to "log"
+    --no-parallel             Process files in parallel
+                              Defaults to "true"
+    --parallel-workers <int>  Number of parallel workers to use
+                              Defaults to "0"
+    --require-pragma          Require either "@prettier" or "@format" to be present in the file's first docblock comment in order for it to be formatted
+                              Defaults to "false"
+    --stdin-filepath <path>   Path to the file to pretend that stdin comes from
+    "
+`;
+
+exports[`show usage with --help (write) 1`] = `[]`;
+
+exports[`show version with --version (stderr) 1`] = `""`;
+
+exports[`show version with --version (write) 1`] = `[]`;
+
+exports[`throw error with --check + --list-different (stderr) 1`] = `""`;
+
+exports[`throw error with --check + --list-different (stdout) 1`] = `
+"
+  Incompatible options: "check" and "list-different" cannot be used together
```

Comment:
in prettier, these are in `stderr` rather than `stdout`

is this a bug and should we be doing the same? we call `exit(msg)` but that logs to stdout (via `specialist`). it appears you can `exit(msg, 1)` but i tried that too and still got stdout.

Reply:
It's a bug, I updated those dependencies to fix it, if you rebase you should see that errors are now logged to stderr.